### ISSUE TITLE
Expression unable to parse following directory listing

### ISSUE THREAD

**Issue by hajekj** [icetee/node-ftp#6]
I have a following listing in FTP, the client doesn't seem to be able to parse it.
```
drwxr-S---    3 105207   501            18 Jul 04  2017 .pki
```
**Update:** It appears like the regex here is omitting `S` in the permissions and the directory listing appears to miss the size value - I wasn't able to find any RFC specification whether the size has to be specified.

**Update 2:** I was able to find this: https://files.stairways.com/other/ftp-list-specs-info.txt and http://cr.yp.to/ftp/list/binls.html

**Update 3:** I found this library which seems to handle the parsing quite good: https://github.com/patrickjuchli/basic-ftp, maybe you can take some inspiration?

**Comment by icetee** [comment]
Soon I will look.

**Comment by icetee** [comment]
@hajekj I changed the regexp. Do you have the opportunity to test?

**Comment by hajekj** [comment]
Works fine! Please release an updated version when possible.

**Comment by icetee** [comment]
I've released [v1.0.3](https://github.com/icetee/node-ftp/releases/tag/v1.0.3) the new version. Thanks for submitting the issue.
